server/src/controllers/verifyUser.js

```diff
@@ -0,0 +1,17 @@
+const jwt = require('jsonwebtoken');
+require('env2')('./config.env');
+const getUser = require('../database/queries/getUser');
+
+module.exports = (req, res, next) => {
+  jwt.verify(req.cookies.token, process.env.SECRET_KEY, (err, decoded) => {
+    if (decoded !== undefined) {
+      getUser(decoded.userID).then((users) => {
+        [req.user = null] = users.rows;
```

Comment:
What is this syntax XD? first time seeing it

Reply:
i dont think you need to get the full userdata from the database, since we include important stuff in our cookie, such as his id and his role(is_admin)

Reply:
eslint asked me to use array destructuring
and 'null' is the default value

Reply:
Ohh i didnt know you can destructure into an object key
You can destructure the parameter for easier to understand code